include/net-snmp/agent/mib_modules.h

```diff
@@ -13,8 +13,6 @@ struct module_init_list {
     struct module_init_list *next;
 };
 
-void            add_to_init_list(char *module_list);
```

Comment:
These two should be kept and these declarations should be removed from include/net-snmp/agent/snmp_vars.h instead. There may be more such issues in this pull request.

Reply:
Done, but required two include statements to be added.